game/surface.hh

```diff
@@ -167,19 +167,19 @@ template <GLenum Type> void OpenGLTexture<Type>::draw(Dimensions const& dim, Tex
 
 template <GLenum Type> void OpenGLTexture<Type>::drawCropped(Dimensions const& orig, TexCoords const& tex) const {
 	Dimensions dim(
-	  orig.x1() + tex.x1 * orig.w(),
-	  orig.y1() + tex.y1 * orig.h(),
-	  orig.w() * (tex.x2 - tex.x1),
-	  orig.h() * (tex.y2 - tex.y1)
-	);
+				   orig.x1() + tex.x1 * orig.w(),
```

Comment:
this should look like the old example above, with all vars layed out in a3x4 grid.